pkg/controller/endpointslice/topologycache/topologycache.go

```diff
@@ -32,6 +33,13 @@ const (
 	OverloadThreshold float64 = 0.2
 )
 
+// Nodes with these labels set to any value will be excluded from capacity
+// calculations for Topology Hints.
+var excludedLabels = sets.NewString(
+	"node-role.kubernetes.io/control-plane",
```

Comment:
don't we have some `const` for these?

Reply:
There's a const in kubeadm package, but felt strange to introduce a kubeadm dependency here. It looks like everywhere else in k/k this is hardcoded: https://cs.k8s.io/?q=node-role.kubernetes.io%2Fcontrol-plane&i=nope&files=&excludeFiles=&repos=kubernetes/kubernetes

Reply:
Updated, thanks!